Assets/SEE/XR/RadialSelection.cs

```diff
@@ -7,403 +7,449 @@
-                    actions.Add("HighlightEdges");
-                    actions.Add("Back");
+                    actions.AddRange(menuEntries);
+                    numberOfRadialParts = actions.Count();
                     radialMenuTrigger = true;
+                    menuEntries.Clear();
                     subMenuPosition = radialPartCanvas.position;
                     subMenuRotation = radialPartCanvas.rotation;
                 }
-                if (actions[i] == ActionStateTypes.Rotate.Name)
+                else if (actions[i] == "HideAll")
                 {
-                    if (actionObject != null)
-                    {
-                        Destroyer.Destroy(actionObject);
-                    }
-                    actionObject = PrefabInstantiator.InstantiatePrefab("Prefabs/Dial").transform.gameObject;
-                    actionObject.transform.position = handTransform.position;
-                    actionObject.SetActive(true);
+                    HideMode = HideModeSelector.HideAll;
+                    TriggerHideAction();
+                }
+                else if (actions[i] == "HideSelected")
+                {
+                    HideMode = HideModeSelector.HideSelected;
+                    TriggerHideAction();
+                }
+                else if (actions[i] == "HideUnselected")
+                {
+                    HideMode = HideModeSelector.HideUnselected;
+                    TriggerHideAction();
+                }
+                else if (actions[i] == "HideOutgoing")
+                {
+                    HideMode = HideModeSelector.HideOutgoing;
+                    TriggerHideAction();
+                }
+                else if (actions[i] == "HideIncoming")
+                {
+                    HideMode = HideModeSelector.HideIncoming;
+                    TriggerHideAction();
+                }
+                else if (actions[i] == "HideAllEdgesOfSelected")
+                {
+                    HideMode = HideModeSelector.HideAllEdgesOfSelected;
+                    TriggerHideAction();
+                }
+                else if (actions[i] == "HideForwardTransitiveClosure")
+                {
+                    HideMode = HideModeSelector.HideForwardTransitiveClosure;
+                    TriggerHideAction();
+                }
+                else if (actions[i] == "HideBackwardTransitiveClosure")
+                {
+                    HideMode = HideModeSelector.HideBackwardTransitiveClosure;
+                    TriggerHideAction();
+                }
+                else if (actions[i] == "HideAllTransitiveClosure")
+                {
+                    HideMode = HideModeSelector.HideAllTransitiveClosure;
+                    TriggerHideAction();
+                }
+                else if (actions[i] == "HighlightEdges")
+                {
+                    HideMode = HideModeSelector.HighlightEdges;
+                    TriggerHideAction();
                 }
```

Comment:
I think we can use `Enum.Parse` here to get rid of some repetition. Specifically, I think we can do something like this:
```csharp
if (actions[i] == "Back") 
{
   // [...] (Keep as-is)
}
else if (Enum.TryParse(actions[i], out HideModeSelector mode))
{
   HideMode = mode;
   TriggerHideAction();
}
else
{
   // [...] (Keep as-is)
}
```